RiotShareExtension/Model/ShareExtensionManager.m

```diff
@@ -412,7 +451,7 @@ - (void)sendText:(NSString *)text toRoom:(MXRoom *)room extensionItem:(NSExtensi
     [self didStartSendingToRoom:room];
     if (!text)
     {
-        NSLog(@"[ShareExtensionManager] loadItemForTypeIdentifier: failed.");
+        NSLog(@"[ShareExtensionManager] sendText: failed.");
```

Comment:
if there is already no text in the beginning of this method, it means that loadItemForTypeIdentifier has failed to give data, no? There is a "send text failed" log in the failureblock of the [MXRoom sendTextMessage...] method